Dockerfile

```diff
@@ -1,74 +1,71 @@
 FROM ubuntu:14.04
```

Comment:
I have tested with Debian:jessie and curiously size of image increased. But it produce a working image as well.